packages/shared/lib/auxiliary/popup/enums/popup-id.enum.ts

```diff
@@ -33,6 +33,7 @@ export enum PopupId {
     RemoveNode = 'removeNode',
     RemoveProposal = 'removeProposal',
     Revote = 'revote',
+    SendFlow = 'sendFlow',
```

Comment:
Not sure why but it feels strange to me that we would have "Flow" in the name... :thinking: